lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -34,6 +34,7 @@ my %URL_SPECIALIZATIONS = (
     'CDBaby'              => qr{^https?://(?:www.)?cdbaby.com/}i,
     'ChangeTip'           => qr{^https?://(?:www.)?changetip.com/tipme/}i,
     'CiNii'               => qr{^https?://(?:www.)?ci.nii.ac.jp/}i,
+    'CPDL'                => qr{^https?://(?:www.)?cdpl.org/wiki/}i,
     'Commons'             => qr{^https?://commons.wikimedia.org/wiki/File:}i,
```

Comment:
No it won't, that is exactly why I crossed it out of my review two days ago. 😉 Neither will `www`. And `d` and `p` are inverted, that is, it does not match anything but misspelled URLs.